src/site/_includes/components/component-preview-tabs/index.ts

```diff
@@ -0,0 +1,89 @@
+function createTabManager(componentPreview: HTMLElement) {
```

Comment:
Extracted out of `src/site/_includes/components/component-preview/index.ts` – no changes to the logic otherwise.